src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -40,6 +40,7 @@ const useGroupInfo = (channelId = null) => {
   const blockModalRef = React.useRef(null);
 
   const [isLoadingMembers, setIsLoadingMembers] = React.useState(false);
+  const [isLoadingAddMember, setIsLoadingAddMember] = React.useState(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `isLoadingAddMember` state is a good approach to handle the loading state when adding members. This will help in providing feedback to the user about the ongoing process.
```diff
+ const [isLoadingAddMember, setIsLoadingAddMember] = React.useState(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->